platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/item/message/WorkflowItemMessageRow.java

```diff
@@ -48,6 +52,9 @@ public DayTime getCreatedAt() {
 
 	public AGUser getCreatedBy() {
 
+		if (isAutoTransitionMessage) {
```

Comment:
This is pretty hacky, but it was the best solution I could come up with after hours of trying to understand the whole message mechanisms and trying different things where nothing worked....